apps/ejabberd/src/mongoose_http_client.erl

```diff
@@ -165,3 +166,8 @@ stop_supervisor() ->
     Proc = sup_proc_name(),
     ok = supervisor:terminate_child(ejabberd_sup, Proc),
     ok = supervisor:delete_child(ejabberd_sup, Proc).
+
+to_binary(P) when is_list(P) ->
+    list_to_binary(P);
+to_binary(P) when is_binary(P) ->
```

Comment:
Yes, unless somebody (like me) puts a binary there. Anyway, does it hurt to have it "binary-safe"?